packages/components/src/components/hds/copy/snippet/index.ts

```diff
@@ -102,23 +132,25 @@ export default class HdsCopySnippetIndexComponent extends Component {
   }
 
   @action
-  onSuccess(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onSuccess(args: any) {
     this.status = 'success';
     this.resetStatusDelayed();
 
-    let { onSuccess } = this.args;
+    const { onSuccess } = this.args;
 
     if (typeof onSuccess === 'function') {
       onSuccess(args);
     }
   }
 
   @action
-  onError(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onError(args: any) {
```

Comment:
same as for the `Copy::Button`
```suggestion
  onError(args: HdsClipboardModifierSignature['Args']['Named']['onError']) {
```